src/App.css

```diff
@@ -1,8 +1,21 @@
+.App {
+  background-color: rgb(245, 246, 250);
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  padding: 30px 10px;
+}
+
 .nav-wrap {
-  background-color: green;
-  margin: 0;
+  background-color: white;
+  width: 100%;
+  height: 100px;
   display: flex;
+  align-items: center;
   justify-content: space-between;
+  border-width: 0 0 2px 0;
+  border-color: rgb(228, 227, 227);
+  border-style: solid;
 }
```

Comment:
- [ ] You should stick to the design as much as possible. I would suggest please make the color, and spacing the same in the design. I would suggest please use the credentials to log in and follow the design. See the image below for reference
- [ ] Please see the logo border that is on the right side and fixes the spacing by seeing at original design
### Your design
![his](https://user-images.githubusercontent.com/81727988/222903439-e6d02dc3-3ae1-42a9-a96f-1babcec1bdac.png)

### Original design
![original](https://user-images.githubusercontent.com/81727988/222903445-8a1abbda-a434-4772-b34e-083b97f14405.png)